workflows/ci-simple/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: argo-hub.ci-simple.0.0.1
+  annotations:
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'This Workflow Template is used to create your first CI pipeline in CSDP'
+    argo-hub/categories: 'argo'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: 'Eti Zaguri'
+    argo-hub/owner_email: 'eti.zaguri@codefresh.io'
+    argo-hub/owner_avatar: 'https://avatars.githubusercontent.com/u/85868206'
+    argo-hub/owner_url: 'https://github.com/eti-codefresh'
+    argo-hub/icon_url: "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/workflows/starting-template/assets/icon.svg"
+    icon-background: "#171A2D"
+spec:
+  #   arguments:
+  # #     List here all the arguments to pass as variables to the Workflow from the Sensor
```

Comment:
why this is commented out? just delete if not needed

Reply:
the idea was to show the possibility to have this here also, i can delete it